app/scripts/services/Session.svc.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+angular.module('yourStlCourts').factory('Session', function () {
+  var latestCitations = null;
+  var lastSelectedCitation = null;
+  function getLatestCitations(citations) {
```

Comment:
Passing citations to a method called `getLatestCitations` seems un-intuitive. Please cleanup logic to explicitly get/set the appropriate citations